net/instaweb/rewriter/cache_extender.cc

```diff
@@ -250,6 +250,24 @@ void CacheExtender::StartElementImpl(HtmlElement* element) {
         driver()->InitiateRewrite(context);
       }
     }
+    HtmlElement::Attribute* datasrcset = element->FindAttribute(HtmlName::kDataSrcset);
+    if (datasrcset != nullptr) {
```

Comment:
Can you call this local variable `data_srcset`? I think that would make it easier to read.

Reply:
Sure